2_Growth-Maturity-Decline.md

```diff
@@ -1,15 +1,33 @@
 # Metric: Growth-Maturity-Decline
 
 
+## Issue Resolution
  Activity Metric | Description
  --- | ---
- Tickets Closed / Tickets Opened | Total number of tickets closed per tickets opened over time
- Bug Time to Close | Time from bug opened to bug closed over time
- Time to Merge | Time from submission to merge or abandon over time
- Review Efficiency | Number of merged patches / Number of abandoned patches over time
- Number of New Contributors | Number of first time contributors over time
- Community Growth | Number of new developers over time
- Community Toxicity | Toxic behavior and language over time
+ Open issues | Total number of open issues.
+ Closed issues | Total number of closed issues.
+ Resolution efficiency | Number of closed issues / number of abandoned issues.
+ Average & median time of open issues | The average & median amount of time issues remain open.
+ Average & median time of first response | The average & median amount of time until the first response to an issue is made.
+ Distribution of open issues over time | Proportional frequency of open issue time duration.
+ Distribution of issue resolution over time | Proportional frequency of closed issue time duration.

```

Comment:
> Distribution of open issues over time | Proportional frequency of open issue time duration.
> Distribution of issue resolution over time | Proportional frequency of closed issue time duration.

I'm not clear what these mean. Is there a different way to express these?

Do we even need them if we assume that all metrics are interesting as a time series? 
That is, could the first two criteria (open issues, closed issues) be duplicates of these last two?

Reply:
There are slight differences. Average and median cover the collective total of all issues, whereas the distribution shows you each individual item. For example, given 10 items, if the average & median both equal 2 days, the distribution could show that 4 issues took 1 day to close, 2 issues took 2 days to close, and 4 issues took 3 days to close.

We could possibly remove the reference to median, and change the distribution items to percentiles. For example:

+ Average time of open issues 
+ Average time of first response
+ Percentile distribution of open issue time
+ Percentile distribution of issue resolution time

This seems a bit more clear to me, so what do you think?

Reply:
Thanks for clarifying. The explanation makes sense. The proposed names are also more clear. I have a hunch that the last two could be expressed more simply - I'll sleep over it.